### PR TITLE
add win ci

### PR DESCRIPTION
**Pull request by asukaminato0721**
add win ci

### CODE REVIEW

**Comment by asukaminato0721** [comment]
idk why it keeps fail, on my linux machine, this works without any problem.

```
env LIBRIME_INCLUDE_DIR=/tmp/tmp/t/dist/include cargo build
   Compiling proc-macro2 v1.0.88
   Compiling glob v0.3.1
   Compiling unicode-ident v1.0.13
   Compiling prettyplease v0.2.24
   Compiling rustix v0.38.37
   Compiling libc v0.2.161
   Compiling linux-raw-sys v0.4.14
   Compiling bitflags v2.6.0
   Compiling cfg-if v1.0.0
   Compiling regex-syntax v0.8.5
   Compiling either v1.13.0
   Compiling memchr v2.7.4
   Compiling minimal-lexical v0.2.1
   Compiling libloading v0.8.5
   Compiling clang-sys v1.8.1
   Compiling bindgen v0.69.5
   Compiling home v0.5.9
   Compiling nom v7.1.3
   Compiling quote v1.0.37
   Compiling itertools v0.12.1
   Compiling syn v2.0.82
   Compiling lazy_static v1.5.0
   Compiling log v0.4.22
   Compiling lazycell v1.3.0
   Compiling shlex v1.3.0
   Compiling rustc-hash v1.1.0
   Compiling regex-automata v0.4.8
   Compiling which v4.4.2
   Compiling cexpr v0.6.0
   Compiling regex v1.11.0
   Compiling librime-sys v0.1.0 (https://github.com/lotem/librime-sys#0c51d64e)
   Compiling t v0.1.0 (/tmp/tmp/t)
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 20.61s
```

**Comment by asukaminato0721** [comment]
@wlh320 mind check it?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -24,3 +24,24 @@ jobs:
...
+          components: clippy
+      - uses: MinoruSekine/setup-scoop@v4.0.1
+        with:
+          buckets: extras
+      - name: Install dependencies
+        shell: bash
+        run: |
+          set -eux -o pipefail
+          scoop bucket add wsw0108 https://github.com/wsw0108/scoop-bucket.git
+          scoop install gcc wget librime
```
[the remark this answers is not in the document]
Comment (asukaminato0721) [reply .github/workflows/build.yml]: thx for pointing it out.

**Comment by wlh320** [comment]
It looks good to me now. Thank you!

**Pull request merged**